client.go

```diff
@@ -246,24 +264,28 @@ func (client *NativeClient) Shell(args ...string) error {
 		ssh.ECHO: 1,
 	}
 
-	fd := os.Stdin.Fd()
+	fd := int(os.Stdin.Fd())
 
-	if term.IsTerminal(fd) {
```

Comment:
Okay. I thought that golang library has support to windows terminal. But I'am wrong. I am going to restore term library.